app/src/main/java/org/oppia/android/app/drawer/NavigationDrawerFragmentPresenter.kt

```diff
@@ -240,7 +239,12 @@ class NavigationDrawerFragmentPresenter @Inject constructor(
             fragment.childFragmentManager.beginTransaction().remove(previousFragment).commitNow()
           }
           val dialogFragment = ExitProfileDialogFragment
-            .newInstance(isFromNavigationDrawer = true)
+            .newInstance(
+              isFromNavigationDrawer = true,
+              isAdministratorControlsSelected =
+                getFooterViewModel().isAdministratorControlsSelected.get() ?: false,
+              lastCheckedItemId = previousMenuItemId ?: -1
```

Comment:
`lastCheckedItemId = previousMenuItemId!!` does work, please check.

Reply:
Shouldn't we avoid using ```!!```?

Reply:
Yes! We should avoid mentioning !!